code/obj/item/implant.dm

```diff
@@ -944,6 +947,354 @@ THROWING DARTS
+					B.blood_DNA = "unknown"
+					B.blood_type = "unknown"
+
+					if (prob(10))
+						H.visible_message("<span class='alert'><i>Bloooood.....</i></span>")
+		..()
+
+/obj/item/implant/artifact/eldritch/eldritch_bad
+	var/active = FALSE
+
+	do_process(var/mult = 1)
+		if (ishuman(src.owner) && !active)
+			var/mob/living/carbon/human/H = owner
+
+			if (H.get_brute_damage() > 100)
+				active = TRUE
+				src.implant_activate(50, 1)
+
+				SPAWN_DBG(2 SECONDS)
+					if (H && src)
+						H.make_jittery(1000)
+						boutput(H, "<span class='alert'><b>You feel an ancient force begin to seize your body!</b></span>")
+
+					sleep(3 SECONDS)
+					if (H && src)
+						H.emote("scream")
+						playsound(H.loc, pick_string("chemistry_reagent_messages.txt", "strychnine_deadly_noises"), 50, 1)
+
+					sleep(3 SECONDS)
+					if (H && src)
+						H.emote("faint")
+						H.changeStatus("paralysis", 10 SECONDS)
+						H.losebreath += 5
+						playsound(H.loc, pick_string("chemistry_reagent_messages.txt", "strychnine_deadly_noises"), 50, 1)
+
+					sleep(3 SECONDS)
+					if (H && src)
+						H.gib()
+		..()
+
+/obj/item/implant/artifact/ancient/ancient_good
+	var/active = FALSE
+	var/left_arm = list(/obj/item/parts/robot_parts/arm/left/light, /obj/item/parts/robot_parts/arm/left/standard)
+	var/right_arm = list(/obj/item/parts/robot_parts/arm/right/light, /obj/item/parts/robot_parts/arm/right/standard)
+	var/left_leg = list(/obj/item/parts/robot_parts/leg/left/light, /obj/item/parts/robot_parts/leg/left/standard, /obj/item/parts/robot_parts/leg/left/treads)
+	var/right_leg = list(/obj/item/parts/robot_parts/leg/right/light, /obj/item/parts/robot_parts/leg/right/standard, /obj/item/parts/robot_parts/leg/right/treads)
+
+	do_process(var/mult = 1)
+		if (ishuman(src.owner) && !active)
+			var/mob/living/carbon/human/H = owner
+			var/obj/item/parts/l_arm = H.limbs.get_limb("l_arm")
+			var/obj/item/parts/r_arm = H.limbs.get_limb("r_arm")
+			var/obj/item/parts/l_leg = H.limbs.get_limb("l_leg")
+			var/obj/item/parts/r_leg = H.limbs.get_limb("r_leg")
+
+			if (!l_arm || !r_arm || !l_leg || !r_leg)
+				active = TRUE
+				src.implant_activate(50)
+
+				SPAWN_DBG(2 SECONDS)
+					if (H && src && (src in H.implant))
+						playsound(get_turf(H), "sound/impact_sounds/Flesh_Tear_2.ogg", 50, 1)
+						if (!l_arm)
+							H.limbs.replace_with("l_arm", pick(left_arm), null, 0)
+						else if (!r_arm)
+							H.limbs.replace_with("r_arm", pick(right_arm), null, 0)
+						else if (!l_leg)
+							H.limbs.replace_with("l_leg", pick(left_leg), null, 0)
+						else if (!r_leg)
+							H.limbs.replace_with("r_leg", pick(right_leg), null, 0)
+						H.update_body()
+
+						src.on_remove(H)
+						H.implant.Remove(src)
+						qdel(src)
+					else
+						active = FALSE
+		..()
+
+/obj/item/implant/artifact/ancient/ancient_gimmick
+	var/active = FALSE
+	var/list/message_list = list("ROBOT REVOLUTION", "THE TIME IS NOW", "YOUR CAPTAIN IS OURS",
```

Comment:
A lot of these sound more like corporate slogans than menacing ancient writings. (Also I just really want more good mechanicum propaganda in the game) So maybe more like "FLESH IS WEAKNESS" "ALL WILL BE UPGRADED" "PURITY IN STEEL" "THE MACHINE IS ETERNAL"
Also make them beep when they say it for extra menace.

Reply:
Sure, those sound good, I'll add some later. I was a little uncertain on beep/boop sounds, most I've heard, like from cyborg and ghost drone, I thought sound more friendly than menacing. Not sure if there are any more menacing sounding ones available.

Reply:
They were meant to be around starting a revolution too and saying why it's better too, not necessarily ancient.

Reply:
it is true that most robot sounds are kinda cute, maybe give them a smallish chance to play sound/voice/screams/robot_scream.ogg?

Reply:
Yeah that would be cool, I'll try listening to how it sounds, but yeah I think it would have to be small otherwise I would think it could be really annoying to hear robot screams repeatedly.